services/airflow/dags/datenspende_vitaldata/data_update/transform.py

```diff
@@ -0,0 +1,27 @@
+import polars
+import ramda as R
+
+from database import camel_case_to_snake_case
+
+
+@R.curry
+def replace(mapping: dict, elements: list) -> list:
+    return [mapping.get(n, n) for n in elements]
+
+
+@R.curry
+def transform_columns(column_mapping: dict, data: polars.DataFrame) -> polars.DataFrame:
+    data.columns = R.pipe(R.map(camel_case_to_snake_case), replace(column_mapping))(
+        data.columns
+    )
+    return data
```

Comment:
This is also duplicate and should ideally also be moved into a shared folder in the datenspende context, so that both the survey and vital data tasks can use it.